inference-engine/src/multi_device/multi_device_exec_network.cpp

```diff
@@ -280,6 +293,7 @@ void MultiDeviceExecutableNetwork::ScheduleToWorkerInferRequest(Task inferPipeli
         } else {
             // _acceleratorDevice could be the same as _cpuDevice, such as AUTO:CPU
             if (_alreadyActualNetwork) {
+                WaitActualNetworkReady();
```

Comment:
how this relates to log introduced in this PR?

Reply:
pls close on this item, this seems to be a logic change unrelated to the logging

Reply:
yes, I know it's confused to add this code. I want to print the information of GPU here, when switch from CPU to GPU.
I also thought if could add the logs in the loading thread, but GPU may load failed [8006](https://github.com/openvinotoolkit/openvino/pull/8006), so add logs in "WaitActualNetworkReady" maybe the best way.

Reply:
add comments in code

Reply:
why it cannot be print when the network loading is completed? (in the async task)

Reply:
We would like to print full logic path include QueryNetwork success and LoadNetwork fail.

Reply:
having a query/print code in the innermost scheduling loop is not acceptable.
from what I see it does always query the config, but just doesn't print that when the LOG_LEVEL is not high enough.

alos the configs are static, so not sure why we need to print them from the scheduling code

Reply:
move the print into load task